test/Microsoft.AspNetCore.Sockets.Tests/WebSocketsTests.cs

```diff
@@ -98,48 +98,6 @@ public class WebSocketsTests
             }
         }
 
-        [Theory]
-        [InlineData(WebSocketMessageType.Text)]
-        [InlineData(WebSocketMessageType.Binary)]
-        public async Task FrameReceivedAfterServerCloseSent(WebSocketMessageType webSocketMessageType)
```

Comment:
It probably should have been removed when we added a timeout to `ProcessSocketAsync`